Dockerfile-manager.rhel7

```diff
@@ -30,6 +32,8 @@ RUN touch $APPBASEDIR/__init__.py
 
 RUN adduser --gid 0 -d $APPBASEDIR --no-create-home -c 'Red Hat Insights' insights
 
+RUN chown -R insights:0 /tmp/prometheus_multiproc
```

Comment:
Maybe it can be done in one RUN with previous line to optimize docker image layers.

Reply:
I think that transparency here is more important than optimizing the docker image layers.